flake.nix

```diff
@@ -193,9 +224,17 @@
           };
 
           devShells.default = pkgs.mkShell {
-            shellHook = ''
-              ${config.pre-commit.installationScript}
-            '';
+            shellHook =
+              let
+                systemdUkify = pkgs.systemdMinimal.override {
```

Comment:
Would be nice if upstream found a way to expose the binaries properly?

Reply:
I don't know if it makes sense to have `pkgs.systemdUkify`. But I understand that they don't want Python in their normal systemd closure.